packages/apollo-gateway/src/index.ts

```diff
@@ -95,7 +89,7 @@ export class ApolloGateway implements GraphQLService {
       this.createSchema(services);
     }
 
-    return { schema: this.schema, executor: this.executor };
+    return { schema: this.schema!, executor: this.executor };
```

Comment:
can we declare a return type for this function?

Reply:
Done